tests/Unit/Translator.php

```diff
@@ -1,43 +1,83 @@
 <?php
+// phpcs:ignoreFile
 
 declare(strict_types=1);
 
 namespace Yokai\EnumBundle\Tests\Unit;
 
+use Symfony\Component\HttpKernel\Kernel;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
-/**
- * @author Yann Eugoné <eugone.yann@gmail.com>
- */
-class Translator implements TranslatorInterface
-{
+if (Kernel::MAJOR_VERSION >= 6) {
```

Comment:
What is this check about?

Reply:
In Symfony 6, the `TranslatorInterface::trans` method has parameter and return typehint.

The diff might be a bit tricky but it is about creating 2 implementation of the same class, depending on Symfony's version

Reply:
Right, as a matter of fact I was confused by the diff...